ibis/omniscidb/tests/test_client.py

```diff
@@ -106,6 +106,129 @@ def test_union_op(alltypes):
         expr.compile()
 
 
+def test_insert_zero_values(test_table):
+    values = []
+    with pytest.raises(com.IbisInputError):
+        test_table.insert(values=values)
+
+
+def test_insert_values(con, test_table):
```

Comment:
probably you can use `temp_table` https://github.com/ibis-project/ibis/blob/master/ibis/omniscidb/tests/conftest.py#L157

avoid to use a fixed name for a new table .. the tests can be executed in parallel .. so if you have 2 tests with the same name it can result in a conflict.

Reply:
I agree with you regarding to table names and for that what there will no such issues tests should have unique names. If this test would contain one table I could use `temp_table` (thanks for suggestion, @xmnlab). But in the test some tables are used and they have unique names. Thus, I'd keep the current names.